BattleTank/Game1.cs

```diff
@@ -747,6 +747,8 @@ protected override void Update(GameTime gameTime)
                         {
                             sound.PlaySound(Sound.Sounds.KLIK);
                             PlayerOneController = GenericGamepadTankActionProvider.DefaultPlayerOneGamepadProvider;
+                            if (PlayerTwoControleler.Equals(GenericGamepadTankActionProvider.DefaultPlayerTwoGamepadProvider)&&!_isAnyAvailbleGamePad2)
+                                PlayerTwoControleler = KeyboardTankActionProvider.DefaultPlayerTwoKeybordLayout;
```

Comment:
Tak z polskiego ten Controleler xD.

Reply:
:+1:  za spostrzegawczość

Reply:
Dlatego to przeglądanie kodu ma tę zaletę, że u kogoś widzi się więcej błędów. Bo każdy je robi - nie ma człeka co by nie robił, a co dopiero programista :D.